core/docker/Dockerfile

```diff
@@ -11,16 +11,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM registry.access.redhat.com/ubi8/ubi
 
-ENV JAVA_HOME /usr/lib/jvm/zulu11
+ARG BASE_DOCKER_IMAGE=azul/zulu-openjdk:17
+FROM $BASE_DOCKER_IMAGE
+
+ENV JAVA_HOME /usr/lib/jvm/zulu17
+
 RUN \
     set -xeu && \
-    yum -y -q install https://cdn.azul.com/zulu/bin/zulu-repo-1.0.0-1.noarch.rpm && \
-    yum -y -q install python3 zulu11-jdk less && \
-    yum -q clean all && \
-    rm -rf /var/cache/yum && \
-    alternatives --set python /usr/bin/python3 && \
+    apt-get update -q && \
+    apt-get install -y -q less python curl && \
```

Comment:
Where is curl used?

Reply:
Health check inside docker

Reply:
https://github.com/trinodb/trino/blob/master/core/docker/bin/health-check